docs/pages/guides/using-flipper.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Using Flipper
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+
+[Flipper](https://fbflipper.com/) is a platform for debugging React Native projects running on an emulator/simulator or a physical device. It supports projects running on Android and iOS platforms, and is available as a desktop application on macOS, Windows, and Linux.
+
+It offers various features such as a device log viewer, interactive native layout inspector, network inspector, local database inspector, crash reporter and more. You can add more plugins available in the Flipper desktop app.
+
+Debugging your Expo projects with Flipper requires the following:
+
+- An Expo project created with [Development Build](/development/introduction/)
+- [`expo-community-flipper` Config Plugin](https://github.com/jakobo/expo-community-flipper)
+
+> **info** The `expo-community-flipper` is based on [Expo's Config plugin](/guides/config-plugins/), which is a system that allows changing Expo config to use native code in your Expo project automatically.
+
+## Prerequisites
+
+Before you get started, make sure you have the following installed on your computer:
+
+- [Flipper desktop app](https://fbflipper.com/)
+- [EAS CLI installed](/build/setup/#1-install-the-latest-eas-cli) and [logged in](/build/setup/#2-log-in-to-your-expo-account) to your Expo account
+
+## Step 1: Run Setup Doctor in Flipper app
+
+Open the Flipper desktop app and click the **Setup Doctor** button from the bottom left menu. This is necessary to ensure that all required dependencies are installed correctly on your computer.
+
+<ImageSpotlight
+  alt="Setup Doctor modal in Flipper."
+  src="/static/images/using-flipper/setup-doctor.jpg"
+  style={{ maxWidth: 720 }}
+/>
+
+If any required dependency is missing, follow the instructions provided by Flipper in the desktop app to install it.
+
+## Step 2: Install expo-dev-client
+
+Since Flipper integration depends on native code, you'll need to create a Development Build. To set it up, you'll need to install the `expo-dev-client` library in your project:
+
+<Terminal cmd={['$ npx expo install expo-dev-client']} />
+
+## Step 3: Install expo-community-flipper
+
+The `expo-community-flipper` plugin depends on the `react-native-flipper` library. Run the following command to install both of them:
+
+<Terminal cmd={['$ npx expo install expo-community-flipper react-native-flipper']} />
+
+## Step 4: Add the Config Plugin
+
+In the **app.json**, add `expo-community-flipper` to the `plugins` array:
+
+```json app.json
+{
+  "plugins": ["expo-community-flipper"]
+}
+```
+
+## Step 5: Configure the Development Build
+
+> **info** If you have already configured EAS and a build profile, you can skip this step. Make sure to run the build command to apply changes from `expo-community-flipper` plugin to your Development Build.
```

Comment:
```suggestion
```
If people already have this set up, I think they can skip this step themselves. Otherwise, we'd need to add a callout to all these instructions saying "if you already did this, you can skip this step"